src/github-issues-prs.ts

```diff
@@ -93,6 +99,61 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 		}
 	}
 
+	private async createIssue() {
+		let remotes: GitRemote[];
+
+		try {
+			remotes = await this.getGitHubRemotes();
+		} catch (err) {
+			return false;
+		}
+
+		let urls: RemoteQuickPickItem[] = remotes.map(remote => {
+			let remoteItem: RemoteQuickPickItem = {
+				label: remote.owner + '/' + remote.repo,
+				remote: remote
+			}
+
+			return remoteItem;
+		});
+
+		if (!urls.length) {
+			window.showInformationMessage('There is no remote to get data from!');
+			return;
+		}
+
+		const callback = (selectedRemote: RemoteQuickPickItem|null) => {
+			if (!selectedRemote) {
+				return;
+			}
+			console.log(selectedRemote.remote);
+
+			const github = new GitHub();
+
```

Comment:
Add authentication to support private repositories (doesn't work in all cases, but we can improve that later):
```
				if (remote.username && remote.password) {
					github.authenticate({
						type: 'basic',
						username: remote.username,
						password: remote.password
					});
				}
```